### PR TITLE
Ramp equipment with maximal load and an explanation for Length for mobile ramps.

### PR DESCRIPTION
**Pull request by ue71603**
MaximalLoad added and some exmplanations ons Length

### CODE REVIEW

File: xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentAccess_version.xsd [hunk 1]
```diff
@@ -956,112 +956,117 @@ Rail transport, Roads and Road transport
...
-				</xsd:annotation>
-			</xsd:element>
-			<xsd:element name="SuitableForCycles" type="xsd:boolean" minOccurs="0">
-				<xsd:annotation>
-					<xsd:documentation>Whether equipment is suitable for cycles.</xsd:documentation>
-				</xsd:annotation>
-			</xsd:element>
-		</xsd:sequence>
-	</xsd:group>
+  <xsd:element name="RampEquipment" substitutionGroup="AccessEquipment">
```
Comment (skinkie) [inline xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentAccess_version.xsd:959]: This is not good.

Comment (ue71603) [reply xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentAccess_version.xsd:959]: what do you suggest? redo the change without the indentation problem?

**Review by skinkie — APPROVED** [review]

**Review by Aurige — APPROVED** [review]

File: xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentAccess_version.xsd [hunk 2]
```diff
@@ -1012,7 +1011,12 @@ Rail transport, Roads and Road transport
 		<xsd:sequence>
 			<xsd:element name="Length" type="LengthType" minOccurs="0">
 				<xsd:annotation>
-					<xsd:documentation>Length of ramp in metres.</xsd:documentation>
+					<xsd:documentation>Length of ramp in metres. In the case of a mobile ramp associated with a vehicle this element is used to convey the length the ramp needs to be extended from the vehicle. This indiciates the necessary width on Quay that is needed.</xsd:documentation>
```
Comment (trurlurl) [inline xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentAccess_version.xsd:1014]:
```suggestion
					<xsd:documentation>Length of ramp in metres. In the case of a mobile ramp associated with a vehicle this element is used to convey the length the ramp needs to be extended from the vehicle. This indicates the minimally necessary width of the Quay.</xsd:documentation>
```

File: xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentAccess_version.xsd [hunk 3]
```diff
@@ -1012,7 +1011,12 @@ Rail transport, Roads and Road transport
...
+				</xsd:annotation>
+			</xsd:element>
+			<xsd:element name="MaximumLoad" type="WeightType" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Maximum load in Kilos. Mainly used for mobile ramps.</xsd:documentation>
```
Comment (trurlurl) [inline xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentAccess_version.xsd:1019]:
```suggestion
					<xsd:documentation>Maximum load in kilograms. Mainly used for mobile ramps.</xsd:documentation>
```

**Review by trurlurl — APPROVED** [review]

**Pull request merged**